protocol/communities/manager.go

```diff
@@ -1200,7 +1200,7 @@ func (m *Manager) InviteUsersToCommunity(communityID types.HexBytes, pks []*ecds
 	return m.inviteUsersToCommunity(community, pks)
 }
 
-func (m *Manager) AddMemberToCommunity(communityID types.HexBytes, pk *ecdsa.PublicKey) (*Community, error) {
+func (m *Manager) AddMemberOwnerToCommunity(communityID types.HexBytes, pk *ecdsa.PublicKey) (*Community, error) {
```

Comment:
This name sounds a little strange
```suggestion
func (m *Manager) AddOwnerToCommunity(communityID types.HexBytes, pk *ecdsa.PublicKey) (*Community, error) {
```